src/css/components/_progress.scss

```diff
@@ -36,7 +40,7 @@
 
 // This increases the size of the progress holder so there is an increased
 // hit area for clicks/touches.
-.video-js .vjs-progress-control:hover .vjs-progress-holder {
+.video-js .vjs-progress-control:hover .vjs-progress-holder.enabled {
```

Comment:
do we need an `enabled` class? Can that just be the default and also have a `disabled` class?

Reply:
sure we can do that